infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -57,7 +57,7 @@
                     + "[-k <seqcache>] [-T <threads>] [-B <size>] [-i <channel-implementation>] "
                     + "[-H <seconds>] [-I <cluster-id>] [-x <ciphers>] [-z <tls-protocols>]] "
                     + "[--metrics] [--metrics-port <metrics_port>]"
-                    + "[--snapshot-batch=<batch-size>] [--lock-lease=<lease-duration>]"
+                    + "[--snapshot-batch=<batch-size>] [--log-replication-config=<log-replication-config-file-path>]"
```

Comment:
This is intended to replace the --plugin as well, right? so we can remove the plugin...

Reply:
I think it's better to have them separately right? Plugin is designed to pass in the jar plugin info. Config is designed for configuring different parameters for log replication.

Reply:
Ok, well I had understood in the meeting that Sundar suggested to keep them in one file so we have less files. I think it's ok, we only need to keep this in mind then so @hisundar makes the proper changes to include this in the init script when this gets merged and we include this new file in the appliance as well (either way we need to include this new file)